.tool-versions

```diff
@@ -0,0 +1,2 @@
+coursier   latest
```

Comment:
coursier is?

Reply:
Something used by the pre-commit hook... I would ideally want the pre-commit hooks to be independent though